sdk/python/lib/pulumi/runtime/resource.py

```diff
@@ -343,7 +382,8 @@ def do_rpc_call():
 
             resp = await asyncio.get_event_loop().run_in_executor(None, do_rpc_call)
         except Exception as exn:
-            log.debug(f"exception when preparing or executing rpc: {traceback.format_exc()}")
+            log.debug(
+                f"exception when preparing or executing rpc: {traceback.format_exc()}")
```

Comment:
Did the linger start complaining about these?  It’s very odd to have to format calls like this.

Reply:
Yeah, my vscode linter was complaining.  And doing a format-doc produced this.  So i just kept it to keep it quiet.  I'd be happy to disable virtually all linting here fwiw.

Reply:
Let me guess, hit 80cols? That's the default specified in [PEP8](https://www.python.org/dev/peps/pep-0008/) but tbh, most developers disable that rule or set it to 100 or 120.

Aside, my personal preference for this is to do the parens like C-style braces and always include commas:
```
log.debug(
    f"exception when preparing or executing rpc: {traceback.format_exc()}",
)
```

This is much more of a thing with bigger blocks, especially nested ones. Again, personal preference and project policies should override.